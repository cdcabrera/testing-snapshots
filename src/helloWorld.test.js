import Hello from './helloWorld';

describe("HelloWorld", () => {
  it('should render a string of HTML', () => {
    expect(Hello.world()).toMatchSnapshot();
  });
});
