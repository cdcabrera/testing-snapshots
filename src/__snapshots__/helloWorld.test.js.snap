// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HelloWorld should render a string of HTML 1`] = `"<strong>Hello World</strong>"`;
